sdk/src/main/java/com/ingresse/sdk/model/response/UserDataJSON.kt

```diff
@@ -18,7 +18,7 @@ data class UserDataJSON(
         val state: String? = "",
         val zip: String? = "",
         val verified: Boolean? = false,
-        val pictures: Any? = null,
+        val pictures: UserPicturesJSON? = null,
```

Comment:
Sometimes we receive pictures as array and sometimes as an object, that's why is **Any** type. Are you sure that with some users this will not crash?